frontend/src/router.js

```diff
@@ -225,14 +216,19 @@ export default new Router({
           }
         },
         {
-          path: '',
+          path: 'dashboard',
           name: 'camp/dashboard',
           component: () => import(/* webpackChungName: "camp" */ './views/camp/Dashboard.vue')
+        },
+        {
+          path: '',
+          name: 'camp/home',
+          redirect: { name: 'camp/dashboard' }
```

Comment:
Any specific reason for introducing a new route which always redirects?

Reply:
Das war nur dazu da, damit man auch ohne Subroute auf den richtigen view kommt.